examples_tests/61.BlitFilterTest/default_compute_alpha_test.comp

```diff
@@ -0,0 +1,81 @@
+#version 460 core
+
+#include <nbl/builtin/glsl/macros.glsl>
+
+#ifndef _NBL_GLSL_WORKGROUP_SIZE_X_
+#define _NBL_GLSL_WORKGROUP_SIZE_X_ 16
+#endif
+
+#ifndef _NBL_GLSL_WORKGROUP_SIZE_Y_
+#define _NBL_GLSL_WORKGROUP_SIZE_Y_ 16
+#endif
+
+#ifndef _NBL_GLSL_WORKGROUP_SIZE_Z_
+#define _NBL_GLSL_WORKGROUP_SIZE_Z_ 16
+#endif
```

Comment:
same as for the normalization, you want different XYZ sizes depending on dimensionality, but always the product to be 256

Reply:
Resolved.